meson.build

```diff
@@ -316,7 +318,7 @@ test('lms', lmst)
 miqrt = executable('miqrt', 'src/miqr/miqrt.f90', link_with : libgalahad)
 test('miqr', miqrt)
 
-mopt = executable('mopt', 'src/mop/mopt.f90', link_with : [libgalahad_blas, libgalahad], dependencies : blas_dep)
+mopt = executable('mopt', 'src/mop/mopt.f90', link_with : libgalahad_link_with + libgalahad, dependencies : blas_dep)
```

Comment:
I'm not sure to understand this line. If we want to use `libopenblas` instead of `libgalahad_blas` and `libgalahad_lapack`, we still need to link `mopt` with it?

Reply:
In that case, `libgalahad_blas/lapack` don't appear in `libgalahad_link_with`, but `openblas` appears in `blas_dep`. With Meson, `link_with` is reserved for targets that we build ourselves, while `dependencies` are for external libraries. If we build our own BLAS, then `blas_dep` is empty.

Reply:
Perfect, thanks for the clarification!